reference/catalog.v3.yml

```diff
@@ -12356,7 +12356,7 @@ paths:
       tags:
         - Product Metafields
       summary: Update a Product Metafield
-      description: "Updates a *Product Metafield*.\n\n**Required Fields**\n* none\n\n**Read-Only Fields**\n* id\n* These fields can only be modified by the app (API credentials) that created the metafield:\n\t* namespace\n\t* key\n\t* permission_set\n\n**Usage Notes**\n* Attempting to modify `namespace`, `key`, and `permission_set` fields using a client ID different from the one used to create those metafields will result in a 403 error message. "
+      description: "Updates a *Product Metafield*.\n\n**Required Fields**\n* none\n\n**Read-Only Fields**\n* id\n* These fields can only be modified by the app (API credentials) that created the metafield:\n\t* `namespace`\n\t* `key`\n\t* `permission_set`\n\t* `value`\n\n**Usage Notes**\n* Attempting to modify `namespace`, `key`, `permission_set`, and `value` fields using a client ID different from the one used to create those metafields will result in a `403` error message. "
```

Comment:
```suggestion
      description: "Updates a *Product Metafield*.\n\n**Required Fields**\n* none\n\n**Read-Only Fields**\n* id\n* These fields can only be modified using the API account that created the metafield:\n\t* `namespace`\n\t* `key`\n\t* `permission_set`\n\t* `value`\n\n**Usage Notes**\n* Attempting to modify the `namespace`, `key`, `permission_set`, or `value` field using an API account different from the one used to create those metafields will result in a `403` error message. "
```

Reply:
the ambiguity here is that the credential most often passed in the API request is the `access_token`. For store API credentials, the `access_token` doesn't change. For app API credentials, the `access_token` can change, but the client ID doesn't change. Because the client ID will remain the same, the app can still successfully use this endpoint with a new `access_token`. But it's confusing because the client ID doesn't actually get sent in the API request.

Reply:
Thanks for catching this @slsriehl. I guess this is an older doc that still contained `Client ID`.